sonota.py

```diff
@@ -343,8 +343,8 @@ def on_message(self, message):
 
         if self.setup_completed and self.test and not self.upgrade:
 
-            hash_user1 = self.getFirmwareHash("static/%s" % upgrade_file_user1)
-            hash_user2 = self.getFirmwareHash("static/%s" % upgrade_file_user2)
+            hash_user1 = self.getFirmwareHash(resource_path("static\%s" % upgrade_file_user1))
+            hash_user2 = self.getFirmwareHash(resource_path("static\%s" % upgrade_file_user2))
```

Comment:
Best to make these both a `os.path.join("static", upgrade_file_userX)` so it's cross platform (or change to a `/` as this now also works on Windows)

Reply:
Thank you! you are right, i fixed this with os.path.join